src/V3EmitCSyms.cpp

```diff
@@ -555,6 +556,40 @@ void EmitCSyms::emitSymImpPreamble() {
     }
 }
 
+void EmitCSyms::emitScopeHier(bool destroy) {
+    if (v3Global.opt.vpi()) {
+        string verb = destroy ? "Tear down" : "Set up";
+        string method = destroy ? "remove" : "add";
```

Comment:
Refactored from the constructor.  These two strings are all that change between the constructor and destructor.